src/main/java/uk/gov/companieshouse/ocr/api/common/CallTypeEnum.java

```diff
@@ -13,7 +13,7 @@ private CallTypeEnum(String callTypeValue) {
 
     @Override
     public String getFieldName() {
-        return "callType";
+        return "call_type";
```

Comment:
Could we define it as a constant ?

Reply:
Good shout. One added with the other field names in JsonConstants